spec/spec_helper.rb

```diff
@@ -57,19 +57,16 @@ def vpc_sanitizer(interaction)
   config.ignore_hosts('codeclimate.com') if ENV['CI']
   config.cassette_library_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.root, 'spec/vcr_cassettes')
 
-  # Used to replace the API Key with the placeholder in the saved VCR.
-  config.define_cassette_placeholder('IBM_CLOUD_VPC_API_KEY') { Rails.application.secrets.ibm_cloud_vpc[:api_key] }
```

Comment:
This was added after the fact to workaround [[this bug]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/202/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94L69)